spec/sanitizer_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'yaml'
+require_relative '../lib/forspell/sanitizer'
+
+class TestClass
+  include Forspell::Sanitizer
+
+  def initialize
+    @custom_dictionary = []
+  end
+end
```

Comment:
BTW, this complicated test setup implies, that you don't need sanitizer to be includable -- it does NOT uses anything of the class it is included to. Calling it as `Sanitizer.sanitize(...)` from loader would be almost as easy, but will lessen the coupling of the code.